llvm/test/CodeGen/AMDGPU/hazards-gfx950.mir

```diff
@@ -1223,3 +1223,21 @@ body:             |
     early-clobber renamable $vgpr2 = V_CVT_SCALEF32_SR_PK_FP4_F16_e64 8, killed $vgpr0, 0, killed $vgpr3, 4, killed $vgpr1, killed $vgpr1, 0, implicit $mode, implicit $exec
     S_SETPC_B64_return undef $sgpr30_sgpr31, implicit killed $vgpr0
 ...
+
+---
```

Comment:
This is not a gfx950 hazard, this belongs in a different file

Reply:
Crash is also seen for gfx950: https://godbolt.org/z/dn1eTjoM8

Reply:
But it's not a gfx950 specific hazard, it still doesn't belong here

Reply:
Added a new file